Store absolute page number in count9 via dv2dt/dt2dv and output it in label, prefixing it with 'abs' to distinguish from other '%%Page:' lines.

diff --git a/dosection.c b/dosection.c
index 1ebc163..6551e5c 100644
--- a/dosection.c
+++ b/dosection.c
@@ -121,7 +121,8 @@
 #endif /* ~SHORTINT */
             fflush(stderr);
          }
-         skipover(36);
+         skipover(32);
+         pagenum = signedquad();
          prevptr = signedquad()+1;
          for (k=0; k<pagecopies; k++) {
             if (k == 0) {
diff --git a/output.c b/output.c
index 1114ee7..079c283 100644
--- a/output.c
+++ b/output.c
@@ -1620,7 +1620,7 @@
 #else
          fprintf(bitfile, "%%DVIPSSectionPage: %d\n", pagenum);
       else if (! isepsf)
-         fprintf(bitfile, "%%%%Page: %d %d\n", pagenum, thispage);
+         fprintf(bitfile, "%%%%Page: abs%d %d\n", pagenum, thispage);
 #endif
    }
    linepos = 0;
